server/src/main/resources/application.example.yml

```diff
@@ -40,6 +40,7 @@ seata:
       group: SEATA_GROUP
       username:
       password:
+      context-path:
```

Comment:
Whether need quotation marks？ 

The following two are not uniform:
server/src/main/resources/application.example.yml
script/client/spring/application.yml

Reply:
Done,pls refers to this PR.